saleor/order/models.py

```diff
@@ -111,10 +111,8 @@ def __str__(self):
         return '#%d' % (self.id, )
 
     def get_total(self):
-        try:
-            return super(Order, self).get_total()
-        except AttributeError:
-            return Price(0, currency=settings.DEFAULT_CURRENCY)
+        # For backwards compatibility
+        return self.total
```

Comment:
BTW, this function is actually required for a proper implementation of `ItemSet`. Therefore it's more than a backwards compatibility wrapper. I suggest we change it to return `self.total` but keep the code calling `order.get_total()`.